src/hearings/containers/view-hearing/hearing-completed-summary/hearing-completed-summary.component.html

```diff
@@ -0,0 +1 @@
+<exui-hearing-actual-summary></exui-hearing-actual-summary>
```

Comment:
```suggestion
<div class="govuk-grid-row">
  <div class="govuk-grid-column-full">
    <h1 class="govuk-heading-l">
      <span class="govuk-caption-l">{{hearingActualsMainModel?.caseDetails.publicCaseName}}</span>
      Hearing details
    </h1>
    <exui-listing-information-summary></exui-listing-information-summary>
    <exui-hearing-actual-summary></exui-hearing-actual-summary>
  </div>
</div>
```
And build the exui-hearing-actual-summary as atomic as possible.